pkg/storage/tsdb/caching_bucket.go

```diff
@@ -169,6 +171,10 @@ func isTenantsDir(name string) bool {
 	return name == ""
 }
 
+func isTenantDeletionMark(name string) bool {
```

Comment:
Any reason why not adding it to `isMetaFile()` like we did for the block deletion mark?

Reply:
No special reason, I can merge it.